### ISSUE TITLE
Showing current sensor value not working

### ISSUE THREAD

**Issue by felixwelsch** [alexarch21/history-explorer-card#194]
Hi,
I am using the minimal example with the latest version of history-explorer-card and Home Assistant 2023.8.1:

```
type: custom:history-explorer-card
showCurrentValues: true
graphs:
  - type: line
    entities:
      - entity: sensor.temperature
```

And not seeing any value in the legend.

<img width="326" alt="Bildschirmfoto 2023-08-06 um 19 24 27" src="https://github.com/alexarch21/history-explorer-card/assets/28998101/e9086c4d-6287-4fa0-848a-5101a733c1de">

What i am doing wrong?

Thanks in advance

**Comment by dafunkydan** [comment]
Nothing wrong, but can't confirm - same HA Version, same Code, getting the Value in Legend.

- Is this your real code (sensor.temperature is really clean and minimal 😉)? If not, post your actual Code.
- It looks like you're on mobile. Have you tried it on a Desktop Machine as well?

**Comment by felixwelsch** [comment]
Thank you for taking care of me.

Yes that is the actual code, intentionally took a simple variable.

And you are right, the screenshot was taken in the iOS Home Assistant app. However, I saw the same result on Win (Chrome) and Mac (Safari) Desktop.

However, it is now (today) ok in the browsers (Win, Mac, iOS). It can only be seen without values in the Home Assistant app.

So I think it could be a caching problem, since I only updated to the current version of history-explorer-card the day before yesterday. How long has this feature been available? My previous version was from early 2022.
